test/test_linalg.py

```diff
@@ -6196,6 +6196,58 @@ def generate_inputs(num_batches):
                     self.assertRaises(RuntimeError, lambda: torch.bmm(b1.cpu(), b2))
                     self.assertRaises(RuntimeError, lambda: torch.bmm(b1, b2, out=res2.cpu()))
 
+    @onlyCPU
+    @torch.inference_mode()
+    def test_bmm_multithreaded(self, device):
```

Comment:
This file is actually only about `torch.linalg.*` functions. Could you move this test to `test_torch.py`?

Reply:
Why does it make sense for it to be in test_torch?

Reply:
Everything that has no better place goes there. In particular testing for all operators that doesn't fit in any better category goes there.